src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+#!/usr/bin/env python3
+# -- coding: utf-8 --
+#
+############################################################################
+#
+# MODULE:      r.forcircular
+# AUTHOR(S):   Francesco Geri, Sandro Sacchelli
+# PURPOSE:     Evaluation of circular bioconomy level of forest ecosystems
```

Comment:
I find that the description of what the module does provided in the `DESCRIPTION` section of the manual page is clearer than this one. Also, note the typo here and in line 17.

```suggestion
# PURPOSE:     Evaluation of circular bioeconomy level of forest ecosystems
```